app/views/evidence/_form.html.erb

```diff
@@ -7,12 +7,11 @@
       f.input :content,
       input_html: {
         class: 'textile',
-        data: {
-          behavior: 'auto-save',
+        data: textile_form_urls.merge({
+          allow_dropdown: action_name == 'new',
```

Comment:
```suggestion
          allow_dropdown: @evidence.persisted?,
```

I'd really like if we could move `action_name == 'new'` down into the view instead of having to make it a transient variable that gets passed through everything. If we can't though let make it relevant to the resource on hand.